app/components/RandomQuote/RandomQuote.js

```diff
@@ -1,5 +1,6 @@
 // @flow
 
+// $FlowFixMe
```

Comment:
Ahh, yee, someone should look into bumping flow to get these types, since it looks like TS has ETA `inf`. Or @SmithPeder?

Reply:
ETA > `inf` 🙃 Have too much server/wiki things with more Prio lately 😢